README.md

```diff
@@ -1,74 +1,52 @@
-* (publicly) on [Docker Hub](https://hub.docker.com/r/alfresco/alfresco-base-java)
+* [Docker Hub](https://hub.docker.com/r/alfresco/alfresco-base-tomcat) image name: `alfresco/alfresco-base-tomcat`
+* [Quay](https://quay.io/repository/alfresco/alfresco-base-tomcat) image name: `quay.io/alfresco/alfresco-base-tomcat`
 
-To build this image, run the following script:
-
-```bash
-IMAGE_REPOSITORY=alfresco/alfresco-base-tomcat
-(cd java-$JAVA_MAJOR/$DISTRIB_NAME-$DISTRIB_MAJOR && docker build -t java-$JAVA_MAJOR-$DISTRIB_NAME-$DISTRIB_MAJOR .)
-docker build -t $IMAGE_REPOSITORY . \
-  --build-arg DISTRIB_NAME=$DISTRIB_NAME \
-  --build-arg DISTRIB_MAJOR=$DISTRIB_MAJOR \
-  --build-arg JAVA_MAJOR=$JAVA_MAJOR \
-  --build-arg TOMCAT_MAJOR=$TOMCAT_MAJOR \
-  --no-cache
-```
-where:
-* DISTRIB_NAME is centos
-* DISTRIB_MAJOR is 7
-* JAVA_MAJOR is 8 or 11
-* TOMCAT_MAJOR is 8 or 9
-
-## Release
-
-Just push a commit on the default branch including `[release]` in the message to trigger a release on Travis CI.
+Example final image: `alfresco/alfresco-base-tomcat:tomcat9-jre11-centos7`
 
-## Pulling released images
+> If you are using this base image in a public repository, please stick to the DockerHub published image.
 
-Builds are available from [Docker Hub](https://hub.docker.com/r/alfresco/alfresco-base-tomcat)
+### Image pinning
 
-```bash
-docker pull alfresco/alfresco-base-tomcat:$TOMCAT_MAJOR_MINOR_VERSION 
-docker pull alfresco/alfresco-base-tomcat:$TOMCAT_VERSION-java-$JAVA_MAJOR-$DISTRIB_NAME-$DISTRIB_MAJOR
-docker pull alfresco/alfresco-base-tomcat:$TOMCAT_VERSION-java-$JAVA_MAJOR-$DISTRIB_NAME-$DISTRIB_MAJOR-$SHORT_SHA256
-```
+The [pinning suggestions provided in alfresco-base-java](https://github.com/Alfresco/alfresco-docker-base-java/blob/master/README.md#image-pinning) are valid for this image too.
 
-where:
-* DISTRIB_NAME is centos
-* DISTRIB_MAJOR is 7
-* JAVA_MAJOR is 8 or 11
-* TOMCAT_MAJOR_MINOR_VERSION is 8.5 or 9.0
-* TOMCAT_VERSION is 8.5.76 or 9.0.59
-* SHORT_SHA256 is the 12 digit SHA256 of the image as available from the registry
-
-*NOTE*
-The default image with $TOMCAT_MAJOR_MINOR_VERSION as tag uses CentOS 7 and Java 11.
-Tomcat 9 images are available with CentOS 8 and Java 11 only.
+### Minimum volume configuration
 
-The builds are identical to those stored in the private repo on Quay, which also supports build-pinning versions.
+Used as parent image and with the default configuration, ensure the following
+volumes are all specified.
 
 ```bash
-docker pull quay.io/alfresco/alfresco-base-tomcat:$TOMCAT_MAJOR_MINOR_VERSION
-docker pull quay.io/alfresco/alfresco-base-tomcat:$TOMCAT_VERSION-java-$JAVA_MAJOR-$DISTRIB_NAME-$DISTRIB_MAJOR
-docker pull quay.io/alfresco/alfresco-base-tomcat:$TOMCAT_VERSION-java-$JAVA_MAJOR-$DISTRIB_NAME-$DISTRIB_MAJOR-$SHORT_SHA256
+VOLUME [ "/usr/local/tomcat/logs", "/usr/local/tomcat/work", "/usr/local/tomcat/conf/Catalina", "/usr/local/tomcat/temp" ]
```

Comment:
I'm not sure I understand the purpose of that. logs management should be left to the user for him to choose his logging config.
Why would `work` and `temp` need to be volume mounted?
Regarding `conf/Catalina` I can see the attemped goal but there very few wsettings one would ever change here and - for now at least - we recommend people should create a new image from ours if they needs to tweak that.

Reply:
this and the next section is copied as is and it's here from [4 years ago](https://github.com/Alfresco/alfresco-docker-base-tomcat/blame/master/README.md#L74) - I've just moved them around the README

I don't know if these suggestions really provide some value worth documenting here, however at this time there is no volume defined in Dockerfile and I don't really know if the user would expect having such folders in volumes or not

Reply:
can be removed

Reply:
@nicdoye do you know which was the rationale behind this requirement to configure volumes? it's not used in compose or charts so it hardly looks like a requirement but more of a convenience to see files on the local machine when developing

Reply:
I've simplified the paragraph stating that volumes are not really mandatory